cf_xarray/accessor.py

```diff
@@ -979,110 +997,6 @@ def __init__(self, obj):
     def __setstate__(self, d):
         self.__dict__ = d
 
-    def _assert_valid_other_comparison(self, other):
-        flag_dict = create_flag_dict(self._obj)
-        if other not in flag_dict:
-            raise ValueError(
-                f"Did not find flag value meaning [{other}] in known flag meanings: [{flag_dict.keys()!r}]"
-            )
-        return flag_dict
-
-    def __eq__(self, other):
-        """
-        Compare flag values against `other`.
-
-        `other` must be in the 'flag_meanings' attribute.
-        `other` is mapped to the corresponding value in the 'flag_values' attribute, and then
-        compared.
-        """
-        flag_dict = self._assert_valid_other_comparison(other)
-        return self._obj == flag_dict[other]
-
-    def __ne__(self, other):
-        """
-        Compare flag values against `other`.
-
-        `other` must be in the 'flag_meanings' attribute.
-        `other` is mapped to the corresponding value in the 'flag_values' attribute, and then
-        compared.
-        """
-        flag_dict = self._assert_valid_other_comparison(other)
-        return self._obj != flag_dict[other]
-
-    def __lt__(self, other):
```

Comment:
Lets bring these other comparisons back. They coul be useful for quality levels for example